File: Kotlin/alg10.kt
/* ALG10 */
fun main() {
  var X: Int
  var Y: Int
  var N1: Int
  var N2: Int
  X = readLine()!!.toInt()
  Y = readLine()!!.toInt()
  if (X > Y) {
    N1 = Y
    N2 = X
  } else {
    N1 = X
    N2 = Y
  }
  println(N1)
  println(N2)
}

File: C#/alg10.cs
using System;
class ALG10
{
    static void Main()
    {
        int X;
        int Y;
        int N1;
        int N2;
        int.TryParse(Console.ReadLine(), out X);
        int.TryParse(Console.ReadLine(), out Y);
        if (X > Y)
        {
            N1 = Y;
            N2 = X;
        }
        else
        {
            N1 = X;
            N2 = Y;
        }
        Console.WriteLine(N1);
        Console.WriteLine(N2);
    }
}
